recipe/meta.yaml

```diff
@@ -19,35 +12,29 @@ source:
 build:
   number: 2
   skip: true  # [win]
-  features:
```

Comment:
we should probably add run exports too

Reply:
I've never used run exports, what should it look like ?
```
build:
  run_exports:
    - libclp
    - libcbc
    ...
```

Reply:
from https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#export-runtime-requirements, it should look something like:

```yaml
build:
  run_exports:
    - {{ pin_subpackage('libpng', max_pin='x.x') }}
```

Reply:
ok